### PR TITLE
Interactive tutorial

### PR DESCRIPTION
**Pull request by rhornung67**
Hands-on exercises for 1/2 day RAJA tutorial

### CODE REVIEW

**Comment by rhornung67** [comment]
@artv3, @davidbeckingsale, @rchen20, @mdavis36 can we get a review on this PR?  We need to get a release out this week for the tutorial next week and this PR must get merged.  Thanks.

**Comment by rhornung67** [comment]
@davidbeckingsale good suggestion. I will make those changes and push them soon.

File: exercises/tutorial_halfday/ex5_line-of-sight.cpp [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+#include "RAJA/RAJA.hpp"
+
+#include "memoryManager.hpp"
+
+/*
+ *  Exercise #5: The line-of-sight problem.
+ *
+ *  In this exercise, you will use use RAJA scan operations to solve
+ *  the 'line-of-sight'' problem, which is staed as:
```
Comment (rchen20) [inline exercises/tutorial_halfday/ex5_line-of-sight.cpp]: Typo, "stated".

Comment (rhornung67) [reply exercises/tutorial_halfday/ex5_line-of-sight.cpp]: Fixed!

**Comment by rhornung67** [comment]
@davidbeckingsale changed "Exercise" to "EXERCISE" and added "TODO" before it.

**Pull request merged**